schemas/telemetry/pre-account/pre-account.4.parquetmr.txt

```diff
@@ -0,0 +1,61 @@
+message pre-account {
+  required group metadata {
+    required binary documentId (UTF8);
+    required int64 Timestamp;
+    optional binary Date (UTF8);
+    optional binary appBuildId (UTF8);
+    optional binary appName (UTF8);
+    optional binary appUpdateChannel (UTF8);
+    optional binary normalizedChannel (UTF8);
+  }
+  required group application {
+    required binary architecture (UTF8);
+    required binary buildId (UTF8);
+    required binary channel (UTF8);
+    required binary name (UTF8);
+    required binary platformVersion (UTF8);
+    required binary version (UTF8);
+    optional binary displayVersion (UTF8);
+    required binary vendor (UTF8);
+    required binary xpcomAbi (UTF8);
+  }
+  optional binary creationDate (UTF8);
+  optional binary id (UTF8);
+  required group environment {
+    required group settings {
+      required binary locale (UTF8);
+    }
+    required group system {
+      required int64 memoryMB;
+      required group os {
+        required binary name (UTF8);
+        required binary version (UTF8);
+        required binary locale (UTF8);
+      }
+      required group cpu {
+        required int64 speedMHz;
+      }
+    }
+    required group profile {
+      required int64 creationDate;
+      optional int64 firstUseDate;
+    }
+  }
+  required group payload {
+    required binary reason (UTF8);
+    required binary ecosystemClientId (UTF8);
+    optional binary uid (UTF8);
+    required int64 duration;
+    required group scalars (MAP) {
```

Comment:
This is only specified as 'object' in the JSON schema (so it could pass validation there and fail on the transformation here)